premium.php

```diff
@@ -528,53 +528,53 @@ function premium() {
 		}
 		$system->printMessage();
 	}
-	else if(isset($_POST['change_color']) && ($player->forbidden_seal || $player->premium_credits_purchased)) {
+	else if(isset($_POST['change_color']) && $player->canChangeChatColor()) {
 		$color = $system->clean($_POST['name_color']);
 		switch($color) {
```

Comment:
same comment as above - if we move color options to an array defined in `User`, then we can do a single array check here